src/openrct2/ride/Track.h

```diff
@@ -18,21 +18,37 @@ constexpr const uint16_t RideConstructionSpecialPieceSelected = 0x100;
 constexpr const int32_t BLOCK_BRAKE_BASE_SPEED = 0x20364;
 
 using track_type_t = uint16_t;
+using bank_type_t = uint8_t;
+using vangle_type_t = uint8_t;
 
 #pragma pack(push, 1)
 struct rct_trackdefinition
 {
     uint8_t type;
-    uint8_t vangle_end;
-    uint8_t vangle_start;
-    uint8_t bank_end;
-    uint8_t bank_start;
+    vangle_type_t vangle_end;
+    vangle_type_t vangle_start;
+    bank_type_t bank_end;
+    bank_type_t bank_start;
     int8_t preview_z_offset;
     uint8_t pad[2] = {};
 };
 assert_struct_size(rct_trackdefinition, 8);
 #pragma pack(pop)
 
+struct VAngleAndBank
+{
+    vangle_type_t VAngle;
+    bank_type_t Bank;
+};
+bool operator==(const VAngleAndBank& vb1, const VAngleAndBank& vb2)
```

Comment:
Compiling fails because these are not declared first. I guess they should be moved up into the struct?

Reply:
It is actually missing a constexpr in the two comparison functions. I was sure I compiled those before opening this PR, but I guess I messed something up...

Reply:
Just noticed that I had this done on my working area, but I did not commit :facepalm: